app/src/main/kotlin/com/intive/picover/auth/repository/AuthRepository.kt

```diff
@@ -20,7 +20,7 @@ class AuthRepository @Inject constructor(
 	private val bitmapConverter: BitmapConverter,
 ) {
 
-	private val userAvatarReference = storageReference.child("user/${requireUser().uid}")
+	private val userAvatarReference by lazy { storageReference.child("user/${requireUser().uid}") }
```

Comment:
If someone would change it, intentionally or by mistake, to ordinary property (previous form), then all tests will still be green.   
The goal of unit testing is to detect such cases, and prevent introducing the same bug once again.
Conclusion: add unit test

Reply:
Is going to be addressed.

Reply:
Test added